### PR TITLE
Merge release 3.2

### PR DESCRIPTION
**Pull request by sagarkapare**

### CODE REVIEW

File: cdap-app-templates/cdap-data-quality/pom.xml [hunk 1]
```diff
@@ -22,6 +22,7 @@
     <groupId>co.cask.cdap</groupId>
     <artifactId>cdap-app-templates</artifactId>
     <version>3.3.0-SNAPSHOT</version>
+    <version>3.2.1</version>
```
Comment (bdmogal) [inline cdap-app-templates/cdap-data-quality/pom.xml]: revert

File: cdap-app-templates/cdap-etl/cdap-etl-api/pom.xml [hunk 2]
```diff
@@ -24,6 +24,7 @@
     <groupId>co.cask.cdap</groupId>
     <artifactId>cdap-etl</artifactId>
     <version>3.3.0-SNAPSHOT</version>
+    <version>3.2.1</version>
```
Comment (bdmogal) [inline cdap-app-templates/cdap-etl/cdap-etl-api/pom.xml]: revert

**Comment by awholegunch** [comment]
Doc changes LGTM. MD5 Hashes look correct.

File: cdap-app-templates/cdap-data-quality/pom.xml (hunk 1 above)
Comment (sagarkapare) [inline cdap-app-templates/cdap-data-quality/pom.xml]: done.

File: cdap-app-templates/cdap-etl/cdap-etl-api/pom.xml (hunk 2 above)
Comment (sagarkapare) [inline cdap-app-templates/cdap-etl/cdap-etl-api/pom.xml]: done.

**Comment by bdmogal** [comment]
:+1:

**Pull request merged**